src/pages/namespace/Explorer/Workflow/Revisions/Overview/TrafficShaping/index.tsx

```diff
@@ -1,14 +1,14 @@
 import { FC, useEffect, useState } from "react";
 import { Trans, useTranslation } from "react-i18next";
 
-import Button from "../../../../../../design/Button";
-import { Card } from "../../../../../../design/Card";
+import Button from "../../../../../../../design/Button";
```

Comment:
should we use a `~/` path here instead? (also applies to the other imports in this file)